commit f09afc33f0583bd1b86519a99756cbc9b905bd45
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2019-04-03T18:48:53+00:00 (6 months ago)
    
    TOOLS-2233 mailing lists have been migrated from Listbox to Topicbox
    Reviewed by: Isaac Davis <isaac.davis@joyent.com>
    Approved by: Isaac Davis <isaac.davis@joyent.com>

diff --git a/README.md b/README.md
index e9161894..115e3eff 100644
--- a/README.md
+++ b/README.md
@@ -717,11 +717,10 @@ tested this on older platform images?
 You can interact with the SmartOS community in a number of ways. This
 includes:
 
-* The *smartos-discuss* mailing list. Once you [subscribe to the
-  list](https://www.listbox.com/subscribe/?list_id=184463), you can send mail to
-  the list address: smartos-discuss@lists.smartdatacenter.org.
-  The mailing list archives are also [available on the
-  web](https://www.listbox.com/member/archive/184463/=now).
+* The *smartos-discuss*
+  [mailing list](https://smartos.topicbox.com/groups/smartos-discuss).
+  If you wish to send mail to the list you'll need to join, but you can view
+  and search the archives online without being a member.
 
 * In the *#smartos* IRC channel on the [Freenode IRC
   network](https://freenode.net/).
